### DISCUSSION TITLE
How to use JobFunnel as a noob non-techie

### DISCUSSION THREAD

**Discussion by osioke** [PaulMcInnis/JobFunnel#174] [Q&A]
I've followed the download and install instructions, but running `funnel load -s demo/settings.yaml` throws an error on Windows. I am most likely missing something very basic and may not even be your code related, please help.

**Comment by PaulMcInnis** [comment]
hey there we're migrating to uv, so you'll have clearer instructions in the next release.

If you are still stuck feel free to send me an email over at `paulmcinnis99@gmail.com`
